packages/docusaurus-theme-common/src/utils/colorModeUtils.tsx

```diff
@@ -34,7 +34,7 @@ const themes = {
   dark: 'dark',
 } as const;
 
-type Themes = typeof themes[keyof typeof themes];
+export type Themes = typeof themes[keyof typeof themes];
```

Comment:
I personally don't like the name `Themes` because it's a union, not a collection. Maybe rename this as `ColorTheme` or `ColorMode` for uniformity?

Reply:
Agreed , I'm not really sure what to consider as the source of truth for getting current (supported) color themes, so maybe this isn't the fit place. We need to decide which term to use, I like "color mode" or "color scheme" better than "[color] theme".

Reply:
Yes, since the component is called `ColorModeToggle` anyways, let's call this type `ColorMode`

Reply:
Although I don't like using the "theme" term, it's better to be consistent in this case anyway. So even in the ColorModeToggle component I use "theme" term. This may not be very clear, but it is consistent with colorModeUtils for now.

Reply:
Actually, I'm going to use ColorMode instead and refactor a bit the colorModeUtils file (keeping old public apis for retrocompatibility but we should plan to remove them), so it should be consistent in the long term